apps/console/src/features/core/components/access-control/access-control-show.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ import { 
+    Children, 
+    FunctionComponent, 
+    PropsWithChildren, 
+    cloneElement, 
+    isValidElement 
+} from "react";
+import { useAccess } from "react-access-control";
+
+export interface AccessControlShowInterface {
+    when: string | string[];
+    notWhen?: string |string[];
+    fallback?: any;
+    resource?: any;
```

Comment:
Is `resource` supposed to be an `object`? If so better to type it.
```suggestion
    resource?: Record<string, any>;
```

Reply:
Fixed with 7d1364f